capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragment.java

```diff
@@ -65,10 +68,12 @@ private void setListener(@NonNull final CameraFragmentImpl fragmentImpl, @Nullab
                             + "You can set it with CameraFragmentCompat#setListener() or "
                             + "by making the host activity implement the CameraFragmentListener.");
         }
+
+        mCancelListener = cancelListener;
```

Comment:
[Request] I think this is not needed because it's in a private method which always receives the `mCancelListener` so this just boils down to `mCancelListener = mCancelListener`. Also then the method can be renamed back to singular `setListener()`.

Reply:
Good catch, thank you for pointing it out!